lib/simctl.ts

```diff
@@ -120,11 +121,19 @@ export class Simctl implements ISimctl {
 		return devices;
 	}
 
+	public getLog(deviceId: string): any {
+		return this.simctlSpawn("spawn", [deviceId, "log", "stream"]);
+	}
+
 	private simctlExec(command: string, args: string[], opts?: any): any {
 		let result = childProcess.spawnSync("xcrun", ["simctl", command].concat(args), opts);
-		if(result && result.stdout) {
+		if (result && result.stdout) {
 			return result.stdout.toString().trim();
 		}
 		return '';
 	}
+
+	private simctlSpawn(command: string, args: string[], opts?: any): any {
```

Comment:
The return type here can be narrowed to `child_process.ChildProcess` instead of `any`

Reply:
In addition, you can specify `opts?: child_process.SpawnOptions` instead of `opts?: any`

Reply:
Will do.